### PR TITLE
Change the way of using env var

### PR DESCRIPTION
**Pull request by sima-zhu**
The PR is to fix the issue of missing comments:
image
Expect the template pipeline information:
image

### CODE REVIEW

File: eng/pipelines/templates/steps/sync-directory.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@ steps:
           -Project "internal"
           -SourceBranch "${{ parameters.UpstreamBranchName }}-ForTestPipeline"
           -DefinitionId "$(${{repo}}-template-tests-definition-id)"
-          -VsoQueuedPipelines "QueuedPipelines"
+          -VsoQueuedPipelines "QUEUEDPIPELINES"
           -CancelPreviousBuilds $True
           -Base64EncodedAuthToken "$(azuresdk-azure-sdk-devops-build-queuing-pat)"
```
Comment (weshaggard) [inline eng/pipelines/templates/steps/sync-directory.yml:113]: We should update the condition to be upper case as well.

File: eng/common/scripts/Queue-Pipeline.ps1 [hunk 2]
```diff
@@ -114,6 +114,5 @@ if ($VsoQueuedPipelines) {
   }else {
     "[$($resp.definition.name)]($($resp._links.web.href))"
   }
```
Comment (weshaggard) [inline eng/common/scripts/Queue-Pipeline.ps1:116]: We should probably keep this outputting to help with debugging.

File: eng/pipelines/templates/steps/sync-directory.yml (hunk 1 above)
Comment (sima-zhu) [reply eng/pipelines/templates/steps/sync-directory.yml:113]: Done.

Comment (weshaggard) [reply eng/pipelines/templates/steps/sync-directory.yml:113]: I don't see the condition updated for this task.

**Review by benbp — APPROVED** [review]

File: eng/pipelines/templates/steps/sync-directory.yml (hunk 1 above)
Comment (weshaggard) [reply eng/pipelines/templates/steps/sync-directory.yml:113]: See condition on line 116, it still doesn't match the casing.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**